src/components/BuildDetailPage/BuildDetailPage.tsx

```diff
@@ -1,5 +1,325 @@
+              <BuildStatusIcon build={serviceContainerBuild.data} long />
+            </AttributesItem>
+
+            <AttributesItem title="Plain Text Log">
+              <OnceBuildIsFinished buildStatus={serviceContainerBuild.data.status}>
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href={`${webConfigService.getPncUrl()}/builds/${serviceContainerBuild.data.id}/logs/build`}
+                >
+                  view
+                </a>
+              </OnceBuildIsFinished>
+            </AttributesItem>
+
+            <AttributesItem title={buildEntityAttributes.id.title} tooltip={buildEntityAttributes.id.tooltip}>
+              {serviceContainerBuild.data.id}
+            </AttributesItem>
+
+            <AttributesItem title={buildEntityAttributes['user.username'].title}>
+              <Username text={serviceContainerBuild.data.user.username} length={40} />
+            </AttributesItem>
+
+            {/* Build times */}
+            <AttributesItem title={buildEntityAttributes.submitTime.title}>
+              {serviceContainerBuild.data.submitTime && createDateTime({ date: serviceContainerBuild.data.submitTime }).custom}
+            </AttributesItem>
+
+            <AttributesItem title={buildEntityAttributes.startTime.title}>
+              {(buildStatusData[serviceContainerBuild.data.status as BuildStatuses].progress !== 'FINISHED' ||
+                serviceContainerBuild.data.startTime) && (
+                <OnceBuildStarted buildStatus={serviceContainerBuild.data.status}>
+                  {serviceContainerBuild.data.startTime &&
+                    createDateTime({
+                      date: serviceContainerBuild.data.startTime,
+                      includeDateInCustom:
+                        !serviceContainerBuild.data.submitTime ||
+                        !areDatesEqual(serviceContainerBuild.data.submitTime, serviceContainerBuild.data.startTime),
+                    }).custom}
+                </OnceBuildStarted>
+              )}
+            </AttributesItem>
+
+            <AttributesItem title={buildEntityAttributes.endTime.title}>
+              <OnceBuildIsFinished buildStatus={serviceContainerBuild.data.status}>
+                {serviceContainerBuild.data.endTime && (
+                  <>
+                    {
+                      // end (date)time
+                      createDateTime({
+                        date: serviceContainerBuild.data.endTime,
+                        includeDateInCustom:
+                          (!!serviceContainerBuild.data.startTime &&
+                            !areDatesEqual(serviceContainerBuild.data.startTime, serviceContainerBuild.data.endTime)) ||
+                          (!!serviceContainerBuild.data.submitTime &&
+                            !areDatesEqual(serviceContainerBuild.data.submitTime, serviceContainerBuild.data.endTime)),
```

Comment:
I think the dates on the detail page should not be shortened.
On the list, it is actually useful to condense it.